service-clients-gen/src/main/java/com/flipkart/poseidon/serviceclients/generator/ServiceGenerator.java

```diff
@@ -372,6 +404,17 @@ private JInvocation createRequest(ServiceIDL serviceIdl, JCodeModel jCodeModel,
             }
         }
 
+        if (endPoint.isIncludeMetaInfo() && endPoint.getMetaInfoMeta() != null && endPoint.getMetaInfoMeta().isDynamicCommandName()) {
+            if (endPoint.getCommandName() != null && !endPoint.getCommandName().isEmpty()) {
```

Comment:
lets not worry about this, make the condition to come into affect if dynamicCommandName is not required

Reply:
please address this